hbase-protocol-shaded/src/main/protobuf/client/Filter.proto

```diff
@@ -88,10 +88,6 @@ message FilterWrapper {
 message FirstKeyOnlyFilter {
 }
 
-message FirstKeyValueMatchingQualifiersFilter {
```

Comment:
Oh, filter has protos...

Suggest sending an email to the dev list to ask if it is OK to remove the proto defination.

The problem here is that, we guarantee old client could communicate with new server, so here if we remove this proto, old client can not communicate with new server any more if it make use of this filter?

Reply:
Got it!